templates/admin/failedJobs.tpl

```diff
@@ -0,0 +1,48 @@
+{**
+ * templates/admin/failedJobs.tpl
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Failed Jobs index
+ *}
+{extends file="layouts/backend.tpl"}
+
+{block name="page"}
+	<h1 class="app__pageHeading">
+		{translate key=$pageTitle}
+	</h1>
+	<div class="app__contentPanel">
+		<pkp-table
+			:columns="columns"
+			:rows="rows"
+			:description="description"
+			:label="label"
+            :api-url="apiUrl"
+		>
+            <template slot="actions" slot-scope="slotProps">
```

Comment:
If `row` is the only property in `slotProps` that you need, we usually use deconstruction to pull it out:

```html
<template slot="actions" slot-scope="{row}">
    ...
    <pkp-button @click="example(row)">
```

Reply:
If there are more props, you can pull them out with:

```
slot-scope="{row, column, etc}"
```

Reply:
Done . and thanks for the tip .